python/ffsim/qiskit/transpiler_passes/lucj_heavy_hex_preset_pass_manager.py

```diff
@@ -0,0 +1,298 @@
+from __future__ import annotations
+
+import copy
+import warnings
+from typing import Any, Sequence
+
+import rustworkx
+from qiskit.passmanager.flow_controllers import ConditionalController
+from qiskit.providers import BackendV2
+from qiskit.transpiler import (
+    StagedPassManager,
+    generate_preset_pass_manager,
+)
+from qiskit.transpiler.passes import ApplyLayout, VF2PostLayout
+from rustworkx import NoEdgeBetweenNodes, PyGraph
+
+import ffsim
+
+
+def _create_two_linear_chains(num_orbitals: int) -> PyGraph:
+    """In zig-zag layout, there are two linear chains (with connecting qubits
```

Comment:
Each docstring should begin with a one-line summary (see https://peps.python.org/pep-0257/#multi-line-docstrings)